paasta_tools/utils.py

```diff
@@ -556,6 +566,9 @@ def get_env_dictionary(self) -> Dict[str, str]:
         env.update(user_env)
         return {str(k): str(v) for (k, v) in env.items()}
 
+    def get_container_port(self) -> int:
+        return self.config_dict.get("container_port", DEFAULT_CONTAINER_PORT)
+
```

Comment:
Why move this from LongRunningServiceConfig?

My understanding is that we only allocate ports for Marathon+K8s tasks (which both subclass LongRunningServiceConfig) -- e.g. Tron actions don't have ports.

Reply:
I wanted to use get_container_port in InstanceConfig, and it's not possible if it's in LongRunningServiceConfig. I didn't know that Tron doesn't use container_port though. Let me move it back.

Reply:
But we'll have two DEFAULT_CONTAINER_PORT in marathon and k8s. What about just leave it in get_env? I don't think Tron cares about having an extra envvar.

Reply:
hmm, let me make it class variable then.